src/kirby/components/list/list.component.html

```diff
@@ -1,61 +1,58 @@
-<!-- HEADER-->
-<div class="list-header" *ngIf="listHeaderTemplate">
-  <ng-container *ngTemplateOutlet="listHeaderTemplate" ></ng-container>
-</div>
-
-<!-- BODY-->
-<ul kirbyInfiniteScroll (scrollEnd)="onLoadOnDemand()" [disabled]="!isLoadOnDemandEnabled">
-  <ng-container *ngIf="isSectionsEnabled; else normalList">
-    <ng-container *ngFor="let section of groupedItems">
-      <li class="section-header">
-        <ng-container
-          *ngTemplateOutlet="
-            sectionHeaderTemplate;
-            context: { $implicit: section.name }
-          ">
-        </ng-container>
-      </li>
-      <li *ngFor="let item of section.items; let idx = index;" [class.last-in-section]="idx === section.items.length - 1"
+
+<ion-list mode="ios" kirbyInfiniteScroll (scrollEnd)="onLoadOnDemand()" [disabled]="!isLoadOnDemandEnabled">
+  <ion-list-header class="list-header" *ngIf="listHeaderTemplate">
```

Comment:
Just use a tag selector:
```suggestion
  <ion-list-header *ngIf="listHeaderTemplate">
```
CSS:
`ion-list-header {...}`